src/chat.rs

```diff
@@ -604,6 +606,40 @@ impl ChatId {
         Ok(())
     }
 
+    /// Adds message "Messages are end-to-end encrypted" if appropriate.
+    /// This function is called on chat creation.
```

Comment:
```suggestion
    ///
    /// This function is rather slow because it does a lot of database queries,
    /// but this is fine because it is only called on chat creation.
```